src/main/java/cn/nukkit/command/simple/SimpleCommand.java

```diff
@@ -26,6 +31,41 @@ public SimpleCommand(Object object, Method method, String name, String descripti
         this.method = method;
     }
 
+    public static CommandFactory factory(Object object, Method method, String desc, String usage, String[] aliases) {
+        return new CommandFactory() {
```

Comment:
Shouldn't this be a lambda?

Reply:
I tried it as a lambda, and IntelliJ complained when I used the `name` parameter in the block, but I just converted it back to a lambda and is working now.